### DISCUSSION TITLE
Winaero Tweaker

### DISCUSSION THREAD

**Discussion by DaintyDust** [ChrisTitusTech/winutil#2292] [Ideas]
winaero tweaker is a program that can tweak alot of windows settings, and i would love to see this be added to the utils 

https://winaerotweaker.com/

**Comment by Marterich** [comment]
The license of the software says 
> [...]You are not allowed to make any copies or redistribute this software including but not limited to making the software available for download or making this software part of a software CD or any other media compilation[...]

So no, this won't be added

**Reply by esobreira** [reply]
That's pretty sad. I love Winaero!
